core-services/prow/02_config/openshift/external-secrets-operator/_prowconfig.yaml

```diff
@@ -0,0 +1,25 @@
+branch-protection:
+  orgs:
+    openshift:
+      repos:
+        external-secrets-operator:
+          branches:
+            main:
+              protect: true
+            release-0.14:
+              protect: true
+tide:
+  queries:
+  - labels:
+    - approved
+    - jira/valid-reference
+    - lgtm
```

Comment:
we can check if we want to add `qe-approved`.

Reply:
comment incorporated.